scripts/rollup/header.js

```diff
@@ -15,14 +15,23 @@ function getProvidesHeader(hasteFinalName) {
 `;
 }
 
-function getUMDHeader(filename, reactVersion) {
+function getHeader(filename, reactVersion) {
   return `/**
```

Comment:
I wonder if this should have an exclamation mark (like `/**!`) as that signals to most minifiers that it should not be removed when minifying the code.

Reply:
I think it's fine to leave as is, we didn't do this before either.